src/evict/evict_lru.c

```diff
@@ -2148,24 +2159,33 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
         WT_ASSERT(session, evict->ref == NULL);
         if (!__evict_push_candidate(session, queue, evict, ref))
             continue;
-        ++evict;
-        ++pages_queued;
-        ++btree->evict_walk_progress;
 
+walk_statistical:
+        if (!urgent_queued_flag) {
+            ++evict;
+            ++btree->evict_walk_progress;
+            ++pages_queued;
+            pages_queued_memory += page->memory_footprint;
+        } else {
+            ++pages_urgent_queued;
+            pages_urgent_queued_memory += page->memory_footprint;
```

Comment:
Thanks @y123456yz, can you help me understand the benefits of having the average page memory footprint for each incremental walk? Personally, I have never had the need for this information.